### ISSUE TITLE
Issue with slope estimation

### ISSUE THREAD

**Issue by xiyang712** [kvos/CoastSat.slope#33]
Hi kvos：
For slope estimation I used shoreline data from 2000-2023 for prediction and satellites used 'L5,L7,L8,L9',but the estimation was not too good. I searched for the reason and found that the revisit period is 16 days instead of 8 days, is it because of the reason that I chose the data after 2020? But another problem arises, as in figure 4: the range of years selected all have two satellites in operation, I hope you can help me where exactly the problem occurs.
Figure1
![image](https://github.com/kvos/CoastSat.slope/assets/115886880/ea59c971-6604-40bf-8464-e8ef2febadfd)
Figure2
![image](https://github.com/kvos/CoastSat.slope/assets/115886880/ce6a8335-26be-4cf0-9abf-c02712e15813)
Figure3
![image](https://github.com/kvos/CoastSat.slope/assets/115886880/be3e9cee-f228-4cb9-8709-ad0b658b9753)
Figure4
![bcd4337abf0544e7d15b43e88c69d2f](https://github.com/kvos/CoastSat.slope/assets/115886880/8b779493-46eb-4c49-927d-860a5b5c2273)

**Comment by kvos** [comment]
yes something has gone wrong there, can you pls show your shoreline time-series?
